go.mod

```diff
@@ -47,7 +50,7 @@ require (
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
-	github.com/testcontainers/testcontainers-go v0.31.0
+	github.com/testcontainers/testcontainers-go v0.32.0
```

Comment:
This will not work due to https://github.com/Kong/kubernetes-ingress-controller/pull/6301#issuecomment-2215224069